docs/juttle_ecosystem.md

```diff
@@ -0,0 +1,19 @@
+# The Juttle Ecosystem
+
+![Juttle Ecosystem](../docs/images/JuttleEcosystemDiagram.png)
+
+The Juttle Ecosystem consists of the following projects:
+
+* [juttle](https://github.com/juttle/juttle) contains the core bits of the language, including the compiler and Javascript runtime for executing Juttle programs, as well as a simple CLI that enables interactive executions using console-based visualizations.
+
+* [juttle-engine](https://github.com/juttle/juttle-engine) is an execution environment for Juttle, which packages together the following components into a complete development and visualization environment.
+
+* [juttle-service](https://github.com/juttle/juttle-service) is a remote execution engine built on node.js for running Juttle programs in response to REST API requests.
+
+* [juttle-viz](https://github.com/juttle/juttle-viz) is a visualization library based on D3 that can render the results of Juttle programs as streaming charts.
+
+* [juttle-client-library](https://github.com/juttle/juttle-client-library) is a client-side Javascript application that interacts with juttle-service to execute juttle programs and render the results in the browser, including support for programmable input controls.
+
+* [juttle-viewer](https://github.com/juttle/juttle-client-viewer) packages juttle-viz and juttle-client-library.
```

Comment:
juttle-viewer is more than just a packaging -- it's also the actual development application. the client-library is an important part of that but all the actual application logic is in the viewer app itself.

How about "juttle-viewer is a development and presentation application for juttle programs that packages juttle-client-library, juttle-viz, and application logic to select and run juttle programs using a remote juttle engine.